staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_test.go

```diff
@@ -59,10 +59,7 @@ func newStorage(t *testing.T) (customresource.CustomResourceStorage, *etcdtestin
 		&metav1.DeleteOptions{},
 	)
 
-	typer := apiserver.UnstructuredObjectTyper{
-		Delegate:          parameterScheme,
-		UnstructuredTyper: discovery.NewUnstructuredObjectTyper(nil),
-	}
+	typer := unstructuredscheme.NewUnstructuredObjectTyper(parameterScheme, discovery.NewUnstructuredObjectTyper(nil))
```

Comment:
having two NewUnstructuredObjectTyper constructors is pretty confusing... any better name to give the delegating one?